### ISSUE TITLE
Profile selection aborts in master

### ISSUE THREAD

**Issue by wllacer** [archlinux/archinstall#936]
When using guided.py the selection of a profile fails with:
```
 File "/home/werner/projects/archinstall/archinstall/lib/menu/selection_menu.py", line 373, in _select_profile
    profile = select_profile()
  File "/home/werner/projects/archinstall/archinstall/lib/user_interaction.py", line 780, in select_profile
    top_level_profiles = sorted(list(list_profiles(filter_top_level_profiles=True)))
  File "/home/werner/projects/archinstall/archinstall/lib/profiles.py", line 107, in list_profiles
    if Profile(None, profile).is_top_level_profile() is False:
  File "/home/werner/projects/archinstall/archinstall/lib/profiles.py", line 254, in is_top_level_profile
    with self.load_instructions(namespace=f"{self.namespace}.py") as imported:
  File "/home/werner/projects/archinstall/archinstall/lib/profiles.py", line 131, in __enter__
    self.execute()
  File "/home/werner/projects/archinstall/archinstall/lib/profiles.py", line 194, in execute
    self.spec.loader.exec_module(sys.modules[self.namespace])
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/werner/projects/archinstall/archinstall/profiles/xorg.py", line 15, in <module>
    *archinstall.lib.hardware.__packages__,
AttributeError: module 'archinstall' has no attribute 'lib'

```

Culprit is
https://github.com/archlinux/archinstall/blob/389feef035cfbb1bd2c4f8be070fa085c088f151/profiles/xorg.py#L10-L16
the reference in line 15 fails to load.

PR on its way

**Comment by wllacer** [comment]
While testing the solution this one happened
```
Press Enter to continue.
 ! Formatting [BlockDevice(/home/werner/vms/testimage.img, size=8.0GB, free_space=5225kB+5243kB+1032kB, bus_type=None)] in 5....4....3....2....1....
Creating a new partition label on /dev/loop0
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/werner/projects/archinstall/archinstall/__main__.py", line 15, in <module>
    archinstall.run_as_a_module()
  File "/home/werner/projects/archinstall/./archinstall/__init__.py", line 224, in run_as_a_module
    script.execute()
  File "/home/werner/projects/archinstall/archinstall/lib/profiles.py", line 194, in execute
    self.spec.loader.exec_module(sys.modules[self.namespace])
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/werner/projects/archinstall/archinstall/examples/guided.py", line 331, in <module>
    perform_filesystem_operations()
  File "/home/werner/projects/archinstall/archinstall/examples/guided.py", line 187, in perform_filesystem_operations
    with archinstall.Filesystem(drive, mode) as fs:
  File "/home/werner/projects/archinstall/archinstall/lib/disk/filesystem.py", line 38, in __exit__
    raise args[1]
  File "/home/werner/projects/archinstall/archinstall/examples/guided.py", line 188, in perform_filesystem_operations
    fs.load_layout(archinstall.arguments['disk_layouts'][drive.path])
  File "/home/werner/projects/archinstall/archinstall/lib/disk/filesystem.py", line 68, in load_layout
    if not self.parted_mklabel(self.blockdevice.device, "gpt"):
  File "/home/werner/projects/archinstall/archinstall/lib/disk/filesystem.py", line 247, in parted_mklabel
    worked = self.raw_parted(f'{device} mklabel {disk_label}').exit_code == 0
  File "/home/werner/projects/archinstall/archinstall/lib/disk/filesystem.py", line 161, in raw_parted
    if (cmd_handle := SysCommand(f'/usr/bin/parted -s {string}')).exit_code != 0:
  File "/home/werner/projects/archinstall/archinstall/lib/general.py", line 407, in __init__
    self.create_session()
  File "/home/werner/projects/archinstall/archinstall/lib/general.py", line 458, in create_session
    with SysCommandWorker(self.cmd, callbacks=self._callbacks, peak_output=self.peak_output, environment_vars=self.environment_vars, remove_vt100_escape_codes_from_lines=self.remove_vt100_escape_codes_from_lines) as session:
  File "/home/werner/projects/archinstall/archinstall/lib/general.py", line 268, in __exit__
    raise SysCallError(f"{self.cmd} exited with abnormal exit code: {self.exit_code}", self.exit_code)
archinstall.lib.exceptions.SysCallError: ['/usr/bin/parted', '-s', '/dev/loop0', 'mklabel', 'gpt'] exited with abnormal exit code: 256
```
@torxed I suppose it has to do with your last tests (last commit in my local copy is 
```
commit 389feef035cfbb1bd2c4f8be070fa085c088f151 (origin/master, origin/HEAD)
Author: Anton Hvornum <anton@hvornum.se>
Date:   Wed Feb 2 15:01:17 2022 +0100

    Torxed pydantic to dataclass (#935)

```
Will send the PR just in case. Pending more tests

**Comment by Torxed** [comment]
I can't reproduce this issue:

![screenshot](https://user-images.githubusercontent.com/861439/152322087-804fe4be-1ab3-4edd-9b52-28e8f9a912fd.jpg)

**Comment by wllacer** [comment]
It works from python's console,
It works as a standard python program
If fails when executed as a module

This is what I used to test  (saved as test.py)
```python
import archinstall
from pprint import pprint
print(archinstall)
print(archinstall.lib.hardware.__packages__)
```
as standard `python test.py`
```
 python test.py
<module 'archinstall' from '/home/werner/projects/archinstall/archinstall/__init__.py'>
['mesa', 'xf86-video-amdgpu', 'xf86-video-ati', 'xf86-video-nouveau', 'xf86-video-vmware', 'libva-mesa-driver', 'libva-intel-driver', 'intel-media-driver', 'vulkan-radeon', 'vulkan-intel', 'nvidia']
```
as a module (test.py saved in examples) `python -m archinstall --script test`
```
[werner@xerneas archinstall]$ python -m archinstall --script test
<module 'archinstall' from '/home/werner/projects/archinstall/./archinstall/__init__.py'>
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
...
  File "/home/werner/projects/archinstall/archinstall/examples/test.py", line 4, in <module>
    print(archinstall.lib.hardware.__packages__)
AttributeError: module 'archinstall' has no attribute 'lib'

```
There is a subtle difference between the import points ( the `\.\' ) but they are synonyms (at least for the os)

**Comment by Torxed** [comment]
Fixed in #916
